lib/kafka.mli

```diff
@@ -134,9 +134,7 @@ val topic_name : topic -> string
   This id will be passed to the delivery callback of the producer,
   once the message delivered.
 *)
-val produce: topic -> partition -> ?key:string -> ?msg_id:msg_id -> string -> unit
-
-val partition_unassigned: partition
```

Comment:
`partition_unassigned` should be removed from the [comment of the produce method](https://github.com/didier-wenzek/ocaml-kafka/pull/25/files#diff-33ac6cfc499695ce92867ecd06a998deR127)